test/tests/functional/pbs_provisioning.py

```diff
@@ -85,10 +85,16 @@ class TestProvisioningJob(TestFunctional):
     def setUp(self):
         TestFunctional.setUp(self)
         self.momA = self.moms.values()[0]
+        self.serverA = (self.servers.values()[0]).shortname
+        self.hostA = self.momA.shortname
```

Comment:
If the only place these two variables are used is in the test for the skipTest(), then make them local variables, not class variables.

Reply:
hostA is used in the 2 test cases. I made the other variable local